vendor.conf

```diff
@@ -119,15 +120,15 @@ github.com/bsphere/le_go                            7a984a84b5492ae539b79b62fb4a
 
 # gcplogs deps
 golang.org/x/oauth2                                 bf48bf16ab8d622ce64ec6ce98d2c98f916b6303
-google.golang.org/api                               dec2ee309f5b09fc59bc40676447c15736284d78 # v0.8.0
+google.golang.org/api                               c24765c18bb761c90df819dcdfdd62f9a7f6fa22 # v0.20.0
 github.com/golang/groupcache                        869f871628b6baa9cfbc11732cdf6546b17c1298
 go.opencensus.io                                    d835ff86be02193d324330acdb7d65546b05f814 # v0.22.3
 cloud.google.com/go                                 ceeb313ad77b789a7fa5287b36a1d127b69b7093 # v0.44.3
 github.com/googleapis/gax-go                        bd5b16380fd03dc758d11cef74ba2e3bc8b0e8c2 # v2.0.5
 google.golang.org/genproto                          3f1135a288c9a07e340ae8ba4cc6c7065a3160e8
 
 # containerd
-github.com/containerd/containerd                    72cec4be58a9eb6b2910f5d10f1c01ca47d231c0 # v1.5.5
+github.com/containerd/containerd                    ee27cde735e2a27a5e1465116ac98593b78d0a6f # v0.0.0-20210721160646-ee27cde735e2
```

Comment:
Should we consider updating to `v1.6.0-beta.1` ? https://github.com/containerd/containerd/compare/ee27cde735e2a27a5e1465116ac98593b78d0a6f..v1.6.0-beta.1

Reply:
Yes that's the idea, was just doing some tests in this PR to repro with only containerd from #42968.